SportsGoods.App/SportsGoods.App.csproj

```diff
@@ -9,6 +9,7 @@
   <ItemGroup>
     <PackageReference Include="MediatR" Version="12.2.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="8.0.3" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
```

Comment:
No testing nuget packages in this project

Reply:
Done (removed)